test/unit/bin-tessel-2.js

```diff
@@ -2,6 +2,7 @@ var sinon = require('sinon');
 var cli = require('../../bin/tessel-2');
 var controller = require('../../lib/controller');
 var logs = require('../../lib/logs');
+var TESSEL_AUTH_KEY = require('../../lib/tessel/provision.js').TESSEL_AUTH_KEY;
```

Comment:
Yes, that's fair. I wanted to keep `require('tessel')` out of the command parser because I want, at some point in the future, the CLI to work for multiple pieces of hardware and importing a specific hardware folder kind of ruins it. But I suppose we can cross that bridge if/when we get to it.